plugins/woocommerce/patterns/page-coming-soon-default.php

```diff
@@ -17,41 +17,40 @@
 ?>
 
 <!-- wp:woocommerce/coming-soon {"comingSoonPatternId":"page-coming-soon-default","className":"woocommerce-coming-soon-entire-site","style":{"color":{"background":"#bea0f2"}}} -->
-<div class="wp-block-woocommerce-coming-soon woocommerce-coming-soon-entire-site has-background" style="background-color:#bea0f2"><!-- wp:cover {"minHeight":100,"minHeightUnit":"vh","isDark":false,"className":"coming-soon-is-vertically-aligned-center coming-soon-cover","layout":{"type":"default"}} --><div class="wp-block-cover is-light coming-soon-is-vertically-aligned-center coming-soon-cover" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim" style=""></span><div class="wp-block-cover__inner-container"><!-- wp:group {"className":"woocommerce-coming-soon-banner-container","layout":{"type":"default"}} -->
-<div class="wp-block-group woocommerce-coming-soon-banner-container"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"10px","bottom":"14px"}}},"className":"woocommerce-coming-soon-header","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignwide woocommerce-coming-soon-header has-background" style="padding-top:10px;padding-bottom:14px"><!-- wp:group {"align":"wide","layout":{"type":"flex","justifyContent":"space-between","flexWrap":"wrap"}} -->
+<div class="wp-block-woocommerce-coming-soon woocommerce-coming-soon-entire-site has-background" style="background-color:#bea0f2"><!-- wp:cover {"customOverlayColor":"transparent","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","className":"coming-soon-is-vertically-aligned-center coming-soon-cover","style":{"spacing":{"padding":{"top":"0px","bottom":"0px","left":"24px","right":"24px"}}},"layout":{"type":"constrained","wideSize":"1280px"}} -->
```

Comment:
I'm not sure if it's feasible, but could we have all new blocks to inherit the color from coming soon block instead? Currently all new blocks will inherit white color:

<img width="1359" alt="image" src="https://github.com/user-attachments/assets/ec5e4e24-4171-41d2-80e1-1c35c7a6a10c" />

Reply:
Yes, I think it should be possible to do. I'll fix it.

Reply:
Ok, fixed!

https://github.com/user-attachments/assets/6165cf71-c219-47bd-b8b0-1da443fec62e

Note: The issue is that the "Cover" block adds a default color style (white or black, depending on the `isDark` attribute) when the color is not set, so the color setting in the coming soon block is not applied. The block editor doesn't support the `inherit` value for the color setting in the editor UI, so we need to manually set the color to inherit for the Cover block.